File: budgets/budget.py
print("This is a budget calculator. What is your monthly income?") 
income = input() 
print("your monthly income is", income ) 

print("How much is your rent each month? ")
rent = input() 

print("How much is your utilities each month?")
utilities = input() 

print("What is your monthly grocery price?") 
groceries = input() 

print("What is your monthly transportation price?") 
transportation = input() 

expenses = float(rent) + float(utilities) + float(groceries) + float(transportation)
print("your monthly expences are ", expenses)

savings = income*(0.3) 
print("your savings are", savings  (" percent of your monthly income") )

spendingMoney = income*(0.1) 
print("your spending money is", spendingMoney  (" percent of your monthly income") )

Rent = income*(0.2) 
print("your rent are", rent  (" percent of your monthly income") )

Utilities = income*(0.2)
print("your utilities are", Utilities  (" percent of your monthly income") )

Groceries = income*(0.1)
print("your groceries are", Groceries  (" percent of your monthly income") )

Transportation = income*(0.1)
print("your transportation cost is", savings  (" percent of your monthly income") )


Expences = float(income) - float(expenses) 

print("your expences are", Expences ("percent of your monthly income. ") )

print("Thank you for using budget calculator!")

File: budgets/budget.c
#include <stdio.h> 
int main(void) {
    float income, rent, utilities, groceries, transportation, savings, expences, spend;
    float prent, putilites, pgroceries, ptransportation, psavings, pexpences; 
  printf("This is a budget calculator.\n how much do you make a month?\n");
  scanf("%f", &income);
  printf("how much dos your rent cost?\n");
  scanf("%f", &rent);
  printf("how much does your utilities cost?\n");
  scanf("%f", &utilities);
  printf("how much does your groceries cost?\n");
  scanf("%f", &groceries);
  printf("how much does your transportation cost?\n");
  scanf("%f", &transportaion);
 printf("you make %.2f\n", income); 
  savings = income * .2; 
 expences = rent + utilities + groceries + transportation;
  spend = income - expences - savings;
  printf("you make $ %.2f\n" , income);
 printf("your expences are $%.2f\n" , income);
  printf("your savings are  $%.2f\n" , expences);
   printf("your spending money is $%f", spend);
   prent = rent/income *100;
   putilities = utilities/income *100; 
   pgroceries = groceries/income *100;
   ptransporation = transporation/income *100;
   psavings = savings/income *100; 
   pexpences = expences/income *100;
   printf("your rent is %d%% of your income.\n", (int) prent);
    printf("your utilities is %d%% of your income.\n", (int) putilities);
    printf("your groceries is %d%% of your income.\n", (int) pgroceries);
    printf("your transportation is %d%% of your income.\n", (int) ptransportation);
    printf("your savings are %d%% of your income.\n", (int) psavings);
    printf("your expences are %d%% of your income.\n", (int) pexpences);
   return 0;
} 